components/layout/OpenAuthoringSiteForm.tsx

```diff
@@ -158,22 +162,43 @@ const OpenAuthoringSiteForm = ({
           )}`
         ),
       afterSubmitFailed: failedForm =>
-        setStatefulPreviewError(failedForm.getState().submitError),
+        setStatefulError(failedForm.getState().submitError),
```

Comment:
We're currently keeping 2 errors in state: "statefulError", and "interpretedError". Can we move the logic from useEffect here, and just setState the `await interpretError(statefulError)` result here?

Reply:
Also, we only really need to keep `asModal` in state. Alerts are fire and forget.